lib/Websocket/Message.php

```diff
@@ -2,11 +2,13 @@
 
 namespace Aerys\Websocket;
 
+use Amp\ByteStream\IteratorStream;
+
 class Message extends \Amp\ByteStream\Message {
     private $binary = false;
 
     public function __construct(\Amp\Iterator $iterator, bool $binary = false) {
-        parent::__construct($iterator);
+        parent::__construct(new IteratorStream($iterator));
```

Comment:
I guess `Message` should just accept an `InputStream` and the `new IteratorStream` should be done when constructing the object?